app/src/main/kotlin/it/vfsfitvnm/vimusic/Database.kt

```diff
@@ -304,6 +304,9 @@ interface Database {
         }
     }
 
+    @Query("UPDATE Format SET isDownloaded = 1")
+    fun update()
```

Comment:
Does it update all the formats? Where is song id?
Also from the name 'update ' it's not straightforward the it will set downloaded to true under the hood. Can we rename the method to have a better name? For example markdownDownloaded ()

Reply:
I guess, update will apply to all rows if we called it without conditions

Reply:
Right it will. But do we need to do it for all songs in the table? Sorry not "markdown" but "mark"

Reply:
I think only for the song we just downloaded. So why would we mark a song downloaded if we actually didn't downlaod it?

Reply:
why would we check `if (downloadedBytes == contentLength)` for some song, and them mark all songs in the database as downloaded. Seems not logical :)